test/e2e_env/multizone/localityawarelb/locality_aware_multizone_hybrid_gateway.go

```diff
@@ -0,0 +1,187 @@
+package localityawarelb
+
+import (
+	"fmt"
+	"net"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/test/resources/samples"
+	. "github.com/kumahq/kuma/test/framework"
+	"github.com/kumahq/kuma/test/framework/client"
+	"github.com/kumahq/kuma/test/framework/deployments/democlient"
+	"github.com/kumahq/kuma/test/framework/deployments/testserver"
+	"github.com/kumahq/kuma/test/framework/envs/multizone"
+)
+
+func LocalityAwareLBGateway() {
+	const mesh = "locality-aware-lb-gateway"
+	const namespace = "locality-aware-lb-gateway"
+
+	meshLoadBalancingStrategyDemoClient := fmt.Sprintf(`
```

Comment:
nit: name points that affect demo-client but affect all dps that communicate with test-server

Reply:
changed